MediaBrowser.Api/Playback/Hls/DynamicHlsService.cs

```diff
@@ -910,8 +925,20 @@ protected override string GetVideoArguments(StreamState state, EncodingOptions e
             {
                 var keyFrameArg = string.Format(
                     " -force_key_frames:0 \"expr:gte(t,{0}+n_forced*{1})\"",
-                    GetStartNumber(state) * state.SegmentLength,
+                    (GetStartNumber(state) * state.SegmentLength).ToString(CultureInfo.InvariantCulture),
```

Comment:
Don't need to call `ToString`, but the `CultureInfo` should be passed to `string.Format`

Reply:
done